### DISCUSSION TITLE
How to read the OEB?

### DISCUSSION THREAD

**Discussion by openenglishbible** [openenglishbible/Open-English-Bible#336] [Ideas]
There are so many different software ways to read Bibles nowadays, I'm not sure what we should be focusing on.

At the moment, we build

- HTML version for reading online
- PDF for printing
- ePub and Kindle for eBook readers

I also build an Accordance module because that's what I mostly use (though I also use Logos)

What do you read Bibles using? What do you think we should add to the list if we can?

**Comment by omarandlorraine** [comment]
I occasionally use an e-reader, and so the epubs are the most helpful for me, after the htmls.

I also use an android app, but not one with OEB. I don't know if such a thing exists.

**Comment by verify0097** [comment]
I like using e-Sword and theWord.

**Reply by openenglishbible** [reply]
I'll have a look into what input formats they accept

**Reply by openenglishbible** [reply]
I've created an Issue to keep track of it: https://github.com/openenglishbible/Open-English-Bible/issues/367

**Comment by Coburn** [comment]
https://www.stepbible.org/

**Reply by openenglishbible** [reply]
StepBible seems to have imported Sword modules, so it has OEB already but a very old version. It looks a nice piece of software though.

**Reply by openenglishbible** [reply]
STEP are importing modules from the Sword project (not the same as e-Sword), so they have the OEB but a very old version. 

I would like to have a nicer online reading experience for the OEB that covers both close study and also longerform reading, but I haven't found a suitable open source system we can use. Something like ESV's online reader would be really nice: https://www.esv.org/Matthew+1/
